packages/core/src/browser/xhrProxy.ts

```diff
@@ -2,8 +2,8 @@ import { monitor, callMonitored } from '../domain/internalMonitoring'
 import { Duration, elapsed, relativeNow, RelativeTime, ClocksState, clocksNow, timeStampNow } from '../tools/timeUtils'
 import { normalizeUrl } from '../tools/urlPolyfill'
 
-interface BrowserXHR extends XMLHttpRequest {
-  _datadog_xhr: XhrStartContext
+interface BrowserXHR<T extends XhrOpenContext = XhrStartContext> extends XMLHttpRequest {
```

Comment:
`XhrStartContext` as default value seems a bit arbitrary to me, what about not having a default value?

Reply:
yes, ok